test/src/test/java/com/coderanch/test/TestTest.java

```diff
@@ -0,0 +1,51 @@
+package com.coderanch.test;
+
+import org.junit.Test;
+
+public final class TestTest {
+
+    private final ConsistentComparableTest testToTest = new ConsistentComparableTest();
+
+    @Test
+    public void allTests_withConsistentObjects_succeed() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add at least one assertion to this test case. ([squid:S2699](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS2699))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack-parent&pullRequest=11&issues=AWbLCXsMSgN4uCclysov&open=AWbLCXsMSgN4uCclysov)

[ISSUE_KEY]: AWbLCXsMSgN4uCclysov